src/main/java/soot/jimple/ClassConstant.java

```diff
@@ -134,6 +134,24 @@ public Type toSootType() {
     return numDimensions > 0 ? ArrayType.v(baseType, numDimensions) : baseType;
   }
 
+  /**
+   * Gets an internal representation used in Java bytecode
+   */
+  public String toInternalString() {
```

Comment:
Could you please add a bit more documentation on how the internal representation actually looks like? It gets a bit confusing on what to expect from all the "toString" methods here...

Reply:
Fixed